code/game/objects/items/spraypaint.dm

```diff
@@ -0,0 +1,42 @@
+/obj/item/spraypaint
+	name = "spraypaint"
+	desc = "Aerosol paint sealed in a pressurized container."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "spraypaint"
+	matter = list(MATERIAL_PLASTIC = 2, MATERIAL_STEEL = 1)
```

Comment:
You can recycle an almost empty one and just print another one...

Reply:
how should i change it so its not the case? idk how to make it more expensive than it actually is on the autolathe

Reply:
2 plastic and 1 steel... That's a very, *very* low price.

Reply:
changed the prices, thanks